### DISCUSSION TITLE
Is this your official website ACLY ? https://kritaaidiffusion.com/

### DISCUSSION THREAD

**Discussion by LeF0URBE** [Acly/krita-ai-diffusion#1009] [Q&A]
the question is : why is it not linked/mentionned on the official github page alongside the top link in the Readme ?
is this buymeacoffee yours ?

**Comment by Acly** [comment]
No this is fake website with content copied from here (probably AI generated?)

**Reply by LeF0URBE** [reply]
thanks for your clarification :). i'll stay a sponsor here anyway
